.github/CHANGELOG.md

```diff
@@ -2,18 +2,68 @@
 
 <h3>New features since last release</h3>
 
+* `TDMProgram`s can now be compiled and submitted via the API.
+  [(#476)](https://github.com/XanaduAI/strawberryfields/pull/476)
+
 <h3>Improvements</h3>
 
+* `copies` are renamed `shots` inside of `TDMProgram` and is now passed via
+  the `eng.run()` method instead of via the program context.
+  [(#489)](https://github.com/XanaduAI/strawberryfields/pull/489)
+
+  ```pycon
+    >>> with prog.context([1, 2], [3, 4]) as (p, q):
+    ...     ops.Sgate(0.7, 0) | q[1]
+    ...     ops.BSgate(p[0]) | (q[0], q[1])
+    ...     ops.MeasureHomodyne(p[1]) | q[0]
+
+    >>> eng = sf.Engine("gaussian")
+    >>> results = eng.run(prog, shots=3)
+  ```
+
+  Furthermore, `TDMProgram.unrolled_circuit` only contains the single-shot
+  unrolled circuit, while the unrolling including shots is returned by the
+  `TDMProgram.unroll()` method, and is strictly contained to this method
+  (and thus not an attribute of the `TDMProgram` class any longer).
+
+* The `reshape_samples` function is updated to return a dictionary where
+  each key correspond to a spatial mode and the values have shape ``(shots,
+  timebins)``.
+  [(#489)](https://github.com/XanaduAI/strawberryfields/pull/489)
+
+* The `Result.samples` is updated for TDM programs to return samples of
+  shape `(shots, spatial modes, timebins)` instead of `(shots, spatial
+  modes * timebins)`.
+  [(#489)](https://github.com/XanaduAI/strawberryfields/pull/489)
+
+* A sample post-processing function is added that allows user's to move
+  vacuum mode measurements from the first shots to the last shots, and
+  potentially crop out the final shots containing these measurements.
+  [(#489)](https://github.com/XanaduAI/strawberryfields/pull/489)
```

Comment:
```suggestion
* A sample post-processing function is added that allows users to move
  vacuum mode measurements from the first shots to the last shots, and
  potentially crop out the final shots containing these measurements.
  [(#489)](https://github.com/XanaduAI/strawberryfields/pull/489)
```

Potentially also not to needed to be included, since it is more of an internal implementation detail?